packages/sdk/src/vaults/constants.ts

```diff
@@ -86,6 +94,20 @@ export function getSubgraphEndpoint(chainId: ChainId) {
   return endpoint;
 }
 
+/**
+ * Gets the address of the NFT whitelist for a given chain id
+ * @param chainId The chain id
+ * @returns
```

Comment:
Minor, non-blocking, apply if you see it fit.

```suggestion
 * @param {ChainId} chainId The chain id we want to get the deployed address from
 * @returns {string} NFT Whitelist deployed address in a supported chain
```